### PR TITLE
i18n cleanup

### PR DESCRIPTION
**Pull request by JeremyKennedy**
This PR just moves some things around and cleans things up. Our `i18n` files were getting huge, so this splits some things out.

### CODE REVIEW

**Review by MarcoGoC — APPROVED** [review]
do we really need to list all the countries? I see some missing

File: i18n/api/countries/en.ts [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+  { key: 'DEU', text: 'Germany' },
+  { key: 'GHA', text: 'Ghana' },
+  { key: 'GRC', text: 'Greece' },
+  { key: 'GRD', text: 'Grenada' },
+  { key: 'GTM', text: 'Guatemala' },
+  { key: 'GIN', text: 'Guinea' },
+  { key: 'GNB', text: 'Guinea-Bissau' },
+  { key: 'GUY', text: 'Guyana' },
+  { key: 'HTI', text: 'Haiti' },
+  { key: 'VAT', text: 'Holy See' },
```
Comment (MarcoGoC) [inline i18n/api/countries/en.ts:75]: LOL isn't Vatican city?

File: i18n/api/countries/en.ts [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+  { key: 'MMR', text: 'Myanmar' },
+  { key: 'NAM', text: 'Namibia' },
+  { key: 'NRU', text: 'Nauru' },
+  { key: 'NPL', text: 'Nepal' },
+  { key: 'NLD', text: 'Netherlands' },
+  { key: 'NZL', text: 'New Zealand' },
+  { key: 'NIC', text: 'Nicaragua' },
+  { key: 'NER', text: 'Niger' },
+  { key: 'NGA', text: 'Nigeria' },
+  { key: 'PRK', text: "Korea (Democratic People's Republic of)" },
```
Comment (MarcoGoC) [inline i18n/api/countries/en.ts:129]: aka North Korea

File: i18n/api/countries/en.ts (hunk 1 above)
Comment (JeremyKennedy) [reply i18n/api/countries/en.ts:75]: lol I never noticed that, but apparently Holy See is more correct: https://en.wikipedia.org/wiki/List_of_ISO_3166_country_codes

**Pull request merged**